shanoir-ng-datasets/src/main/java/org/shanoir/ng/datasetacquisition/model/DatasetAcquisition.java

```diff
@@ -67,7 +67,7 @@ public abstract class DatasetAcquisition extends AbstractEntity {
 	
 	/** Datasets. */
 	@JsonIgnore
-	@OneToMany(fetch = FetchType.LAZY, mappedBy = "datasetAcquisition", cascade = CascadeType.ALL)
+	@OneToMany(fetch = FetchType.EAGER, mappedBy = "datasetAcquisition", cascade = CascadeType.ALL)
```

Comment:
Hi @julien-louis, thank you for our exchange! Here is another place, where it might be interesting to keep lazy please. With kind regards, Michael